cypresse2e/cypress/integration/e2eTests/privacyPolicy/privacyPolicy.spec.js

```diff
@@ -0,0 +1,261 @@
+import PrivatePolicyPage from "../../../elements/pages/privacyPolicy";
+import privatePolicyContent from "../../../fixtures/privatePolicyContent.json";
+
+describe("Private Policy for FightPandemics", () => {
+  const privacyPolicy = new PrivatePolicyPage();
```

Comment:
PrivatePolicyPage() should match the exported class name exported from the POM file

Reply:
@jessica-lau , Yes it will be updated in next commit.TY